src/main/java/com/iexec/worker/feign/SmsClientWrapper.java

```diff
@@ -0,0 +1,48 @@
+package com.iexec.worker.feign;
+
+import com.iexec.common.sms.SmsSecretRequest;
+import com.iexec.common.sms.SmsSecretRequestBody;
+import com.iexec.common.sms.SmsSecretResponse;
+import com.iexec.common.sms.TaskSecrets;
+
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+
+@Slf4j
+@Service
+public class SmsClientWrapper {
```

Comment:
Same thing again here, *Wrapper is not explicit of what the class does. Let's try not to use this kind of name.

Reply:
Done